<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUpdateDeleteUV</class>
 <widget class="QDialog" name="DialogUpdateDeleteUV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="msgAfterDeletion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="updateUVverticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="code">
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="codeLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="titre">
          <property name="text">
           <string>Titre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titreLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nombre de crédits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="nbCreditspinBox">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="saison">
          <property name="text">
           <string>Saison</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="saisonComb"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="categorie">
          <property name="text">
           <string>Catégorie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="categorieComb"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tshType_2">
    <item>
     <widget class="QLabel" name="tshLabelType_2">
      <property name="text">
       <string>Type TSH</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="tshTypeCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>391</width>
     <height>70</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="isFinalTest_2">
           <property name="text">
            <string>Final</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutFinalCombo_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="isMdianTest_2">
           <property name="text">
            <string>Médian</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutMedianCombo_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="isProjectTest_2">
           <property name="text">
            <string>Projet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutProjCombo_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="tauxDeReussite_2">
         <property name="text">
          <string>Taux de réussite récent </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="probaReussiteUV_2">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="isTP_2">
           <property name="text">
            <string>TP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutTpComb_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="isTD_2">
           <property name="text">
            <string>TD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutTDCombo_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trueUpdateUV">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>160</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="uvsModelCombo"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="deleteUV">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateUV">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>uvsModelCombo</zorder>
  <zorder>deleteUV</zorder>
  <zorder>updateUV</zorder>
  <zorder>msgAfterDeletion</zorder>
  <zorder>layoutWidget</zorder>
  <zorder></zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>tshTypeCombo</zorder>
  <zorder>description</zorder>
  <zorder>splitter</zorder>
  <zorder>label</zorder>
  <zorder>trueUpdateUV</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
